node-repository/src/main/java/com/yahoo/vespa/hosted/provision/archive/ArchiveUriManager.java

```diff
@@ -47,8 +45,15 @@ public Optional<String> archiveUriFor(Node node) {
         return Optional.ofNullable(node.cloudAccount().isEnclave(zone) ?
                 archiveUris.get().accountArchiveUris().get(node.cloudAccount()) :
                 archiveUris.get().tenantArchiveUris().get(app.tenant()))
+                .map(uri -> {
+                    // TODO (freva): Remove when all URIs dont have tenant name in them anymore
+                    String tenantSuffix = app.tenant().value() + "/";
```

Comment:
Use tenantSuffix = **"/" +** app.tenant().value() + "/", to ensure the suffix is actually a path component and doesn't match the random authority suffix?

Reply:
Good point, fixed.